homeassistant/components/actiontec/device_tracker.py

```diff
@@ -51,7 +51,7 @@ def __init__(self, config: ConfigType) -> None:
         self.last_results: list[Device] = []
         data = self.get_actiontec_data()
         self.success_init = data is not None
-        _LOGGER.info("Scanner initialized")
+        _LOGGER.debug("Scanner initialized")
```

Comment:
```suggestion
```
We usually don't want side effects in constructors, and I _think_ logging was one of them